PWGLF/TableProducer/cascqaanalysis.cxx

```diff
@@ -87,6 +89,8 @@ struct cascqaanalysis {
     registry.add("hDCAz_AfterCut", "hDCAz_AfterCut", HistType::kTH2F, {{400, -0.2, 0.2, "DCAz"}, {150, 0.0, 15.0, "p_{T} (GeV/c)"}});
     registry.add("hDCAxy_BefCut", "hDCAxy_BefCut", HistType::kTH2F, {{400, -0.2, 0.2, "DCAxy"}, {150, 0.0, 15.0, "p_{T} (GeV/c)"}});
     registry.add("hDCAxy_AfterCut", "hDCAxy_AfterCut", HistType::kTH2F, {{400, -0.2, 0.2, "DCAxy"}, {150, 0.0, 15.0, "p_{T} (GeV/c)"}});
+    registry.add("hNchMultFT0M", "hNchMultFT0M", HistType::kTH2F, {{150, 0.f, 150.f, "N_{ch}"}, {10000, 0.f, 10000.f, "FT0M signal"}});
+    registry.add("hNchMultFV0A", "hNchMultFV0A", HistType::kTH2F, {{150, 0.f, 150.f, "N_{ch}"}, {10000, 0.f, 10000.f, "FV0A signal"}});
```

Comment:
for the FT0M and FV0A I just inherited the max from O2Physics/Common/Tasks/multiplicityQa.cxx. Nevertheless, it could be more reasonable to have kTProfile in Mult axis to get the average signal in FT0M/FV0A for each multiplicity in the event: that will help with the binning issue (at least for the Mult axis). For the Nch one - idk what to expect in VHM events, so we can put 1k for it if you agree.